app/javascript/packs/add_backend_usage.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import { AddBackendFormWrapper } from 'BackendApis'
+import { safeFromJsonString } from 'utilities/json-utils'
+
+const containerId = 'add-backend-form'
+
+document.addEventListener('DOMContentLoaded', () => {
+  const container = document.getElementById(containerId)
+
+  if (!container) {
+    return
+  }
+
+  const { backends, url, newBackendPath } = container.dataset
+  console.log(container.dataset)
```

Comment:
Do we leave `console.log` ?

Reply:
No, Eslint should have warned about it 🤔